tools/bundler.js

```diff
@@ -1704,14 +1711,16 @@ _.extend(ServerTarget.prototype, {
     var archToPlatform = {
       'os.linux.x86_32': 'Linux_i686',
       'os.linux.x86_64': 'Linux_x86_64',
-      'os.osx.x86_64': 'Darwin_x86_64'
+      'os.osx.x86_64': 'Darwin_x86_64',
+      // XXX once we put Windows dev_bundles to warehouse we should fill these
+      // fields in.
+      'os.windows.x86_32': 'XXX',
+      'os.windows.x86_64': 'XXX'
     };
     var platform = archToPlatform[self.arch];
     if (! platform) {
-      buildmessage.error("MDG does not publish dev_bundles for arch: " +
+      throw new Error("MDG does not publish dev_bundles for arch: " +
```

Comment:
why? I remember it was swallowing some errors but can we still have a correct behavior using `buildmessage`? @glasser recently replaced all random errors like these with buildmessage, probably should consult him on this.